### PR TITLE
Add setup with webpack and mock html

### PR DESCRIPTION
**Pull request by niicode**
In this project, I was able to achieve the following:

- Setup the project with webpack
- Added mockup HTML and styling to the index.html page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,15 +54,9 @@ Feel free to check the [issues page](../../issues/).
...
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+    
+![](https://img.shields.io/badge/stars-0.0.1-brightgreen.svg)
 
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
-
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [x] Please replace `./LICENSE` by `./MIT.md` to make this this link work because this link is broken :sob:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @niicode  :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following GitFlow

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (niicode) [reply README.md]: Thank you I have fixed it

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @niicode 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**